service.py

```diff
@@ -51,11 +50,20 @@ def search(item):
 
     try:
         search_start_time = time.time()
-        if item['tvshow'] != "":
-            subtitles_options = downloader.search_tvshow(item['tvshow'], item['season'], item['episode'])
+        
+        if (item['mansearch'] == False):
+            if item['tvshow'] != "":
+                subtitles_options = downloader.search_tvshow(item['tvshow'], item['season'], item['episode'])
+            else:              
+                title, year = xbmc.getCleanMovieTitle(item['title'])
+                subtitles_options = downloader.search_movie(title)
         else:
-            title, year = xbmc.getCleanMovieTitle(item['title'])
-            subtitles_options = downloader.search_movie(title)
+            item['tvshow'], item['season'], item['episode'] = check_and_parse_if_title_is_TVshow(item['title'])
+            subtitles_options = downloader.search_tvshow(item['tvshow'], int(item['season']), int(item['episode']))
+            if (subtitles_options == None): # This is a movie title
+                item['title'] = item['title'].replace("%20", "%2b") # " " to "+"
```

Comment:
From the tests I did this is required.
Without this line, if user input for example: "Avatar 2009"
item['title'] = Avatar%202009
After getCleanMovieTitle you will get: title = avatar, year = 2009, which is OK but search_movie function return None.
With this line you are getting: item['title'] = Avatar%2b2009
And after getCleanMovieTitle => title = Avatar+2009, year = (empty)
Then for unknown reason you are actually getting results from the search_movie function.